### DISCUSSION TITLE
咨询一下formily后续的演进路线

### DISCUSSION THREAD

**Discussion by Landon-CN** [alibaba/formily#3099] [Q&A 2.x] [answered]
你好，我们团队在中后台的表单场景里，大量使用了formily的解决方案，取得了不错的效果。
我们现在不满足只解决表单这一个场景，想在formily上有更多的建设来提升团队开发效率。所以希望能得到作者对formily next的一些思考和后续演进路线

**Comment by janryWang** [comment, marked as answer]
其实formily上限很高的，它的json-schema协议不只是解决表单，页面也能解决，我们团队就在用formily作为页面渲染引擎，它后续的方向，可能更多的会进一步优化协议，减少一些可有可无的概念，不会有太大变化了，因为中后台场景，好像真的用formily就够了

**Reply by Landon-CN** [reply]
协议本身很强大这个毋庸置疑，但是现在只是聚焦了表单这单个场景。
中后台常见的场景有两种，一种是表单，另一种则是查询。
我们在思考是不是能向antd pro那样能把formily的领域扩展到整个中后台当中，这样我们绝大部分开发任务就可以很快的迭代了

**Reply by AdmireWang** [reply]
请问作为页面渲染引擎，有什么方案？
